scipy/linalg/decomp.py

```diff
@@ -1267,3 +1267,138 @@ def hessenberg(a, calc_q=False, overwrite_a=False, check_finite=True):
     q, info = orghr(a=hq, tau=tau, lo=lo, hi=hi, lwork=lwork, overwrite_a=1)
     _check_info(info, 'orghr (hessenberg)', positive=False)
     return h, q
+
+
+def cdf2rdf(w, v):
+    """
+    Converts complex eigensystems to real eigenvalues in a block diagonal form
```

Comment:
I think *eigensystem* is not the mainstream terminology. I know this is matlab lingo but I don't also know what that actually means. Something like 

     Converts complex eigenvalue/eigenvectors into real block diagonal form such that ... holds where ...

Reply:
I belive *eigenstructure* is used in several numerical books and papers I read about computing eigenvalues and eigenvectors, but I agree that it is best to use *eigenvalues* and *eigenstructure* in the docstring.